runtime/onert/frontend/circle_traininfo/include/traininfo_loader.h

```diff
@@ -27,7 +27,7 @@ namespace train
 namespace traininfo_loader
 {
 
-static constexpr char *const TRAININFO_METADATA_NAME = "CIRCLE_TRAINING";
+static const char *const TRAININFO_METADATA_NAME = "CIRCLE_TRAINING";
```

Comment:
While I was on the next step, I found an error :  
```bash 
ONE/runtime/onert/frontend/circle_traininfo/include/traininfo_loader.h:30:56: 
error: ISO C++ forbids converting a string constant to ‘char*’ [-Werror=write-strings]
   30 | static constexpr char *const TRAININFO_METADATA_NAME = "CIRCLE_TRAINING";
      |                                                        ^~~~~~~~~~~~~~~~~

``` 

It looks like `constexpr` doesn't guarantee that char-array(string) content is `const`.
So, For clearness, Let's just remove `constexpr` and use `const` instead.

-------------------------------------------------------------


I refered stack-overflow https://stackoverflow.com/questions/30561104/const-constexpr-char-vs-constexpr-char.
I tried to find an official C++ document about this, But it's hard to find a clear explanation about this.